pmlc/dialect/pxa/transforms/test_analysis.cc

```diff
@@ -32,7 +32,7 @@ struct TestStrideInfoPass : public TestStrideInfoBase<TestStrideInfoPass> {
   void runOnOperation() final {
     auto op = getOperation();
     op->walk([&](Operation *op) {
-      TypeSwitch<Operation *>(op)
+      llvm::TypeSwitch<Operation *>(op)
```

Comment:
I'm surprised this is actually needed, usually llvm symbols are aliased into the mlir namespace.

Reply:
Yeah, the problem here is that they are both present and it can't decide which one to use.